mkdocs/docs/HPC/torque_frontend_via_jobcli.md

```diff
@@ -0,0 +1,153 @@
+
+For example:
+```shell
+$ qsub --help
+usage: qsub [--version] [--debug] [--dryrun] [--pass OPTIONS] [--dump PATH]...
+
+Submit job script
+
+positional arguments:
+  script_file_path      Path to job script to be submitted (default: read job
+                        script from stdin)
+
+optional arguments:
+  -A ACCOUNT            Charge resources used by this job to specified account
+  ...
+```
+
+#### dryrun option
+
+Adding `--dryrun` to a Torque command when using it on the {{hpcinfra}} will show the user what Slurm commands are generated by that Torque command by jobcli. Using `--dryrun` will not actually execute the Slurm backend command.
+
+See [the examples](./#examples)
+
+#### debug option
+
+Similarly to `--dryrun`, adding `--debug` to a Torque command when using it on the {{hpcinfra}} will show the user what Slurm commands are generated by that Torque command by jobcli. However in contrast to `--dryrun`, using `--debug` will actually submit the job afterwards.
+
+See [the examples](./#examples)
+
+#### Examples
+
+The following examples illustrate the working of the `--dryrun` and `--debug` options with and example jobscript.
+
+`example.sh`:
+
+```shell
+#/bin/bash
+#PBS -l nodes=1:ppn=8
+#PBS -l walltime=2:30:00
+
+module load SciPy-bundle/2023.11-gfbf-2023b
+
+python script.py > script.out.${PBS_JOBID}
+```
+
+##### Example of the dryrun option
+
+Running the following command:
+
+```shell
+$ qsub --dryrun example.sh -N example
+```
+
+will generate this output:
+
+```shell
+
+Command that would have been run:
+---------------------------------
+
+/usr/bin/sbatch
+
+Job script that would have been submitted:
+------------------------------------------
+
+#!/bin/bash
+#SBATCH --chdir="/user/gent/400/{{userid}}"
+#SBATCH --error="/kyukon/home/gent/400/{{userid}}/examples/%x.e%A"
+#SBATCH --export="NONE"
+#SBATCH --get-user-env="60L"
+#SBATCH --job-name="example"
+#SBATCH --mail-type="NONE"
+#SBATCH --nodes="1"
+#SBATCH --ntasks-per-node="8"
+#SBATCH --ntasks="8"
+#SBATCH --output="/kyukon/home/gent/400/{{userid}}/examples/%x.o%A"
+#SBATCH --time="02:30:00"
+
+### (start of lines that were added automatically by jobcli)
+#
+# original submission command:
+# qsub --dryrun example.sh -N example
+#
+# directory where submission command was executed:
+# /kyukon/home/gent/400/{{userid}}/examples
+#
+# original script header:
+# #PBS -l nodes=1:ppn=8
+# #PBS -l walltime=2:30:00
+#
+### (end of lines that were added automatically by jobcli)
+
+#/bin/bash
+
+module load SciPy-bundle/2023.11-gfbf-2023b
+
+python script.py > script.out.${PBS_JOBID}
+```
+This output consist of a few components. For our example the most important lines are the ones that start with `$SBATCH` since these contain the translation of the Torque commands to Slurm commands. For example the job-name is the one we specified with the `-N` option in the command.
```

Comment:
```suggestion
This output consist of a few components. For our example the most important lines are the ones that start with `#SBATCH` since these contain the translation of the Torque commands to Slurm commands. For example the job-name is the one we specified with the `-N` option in the command.